dartdoc_test/lib/src/extractor.dart

```diff
@@ -45,17 +46,24 @@ class DocumentationComment {
   });
 }
 
+/// A code sample extracted from a documentation comment.
 class DocumentationCodeSample {
+  /// The documentation comment that contains the code sample.
   final DocumentationComment comment;
+
+  /// content of the code sample.
   final String code;
-  final List<String> imports;
 
   DocumentationCodeSample({
     required this.comment,
     required this.code,
-    this.imports = const [],
   });
 
+  bool get ignore => code.contains('// dartdoc_test:ignore_error');
```

Comment:
```suggestion
  bool get ignore => code.contains('// ignore:dartdoc_test');
```

Reply:
TODO: investigate if we can look for this outside the _documentation comment_, which might be very hard :D